### PR TITLE
Fix StandaloneImage.save() md5 hash generation

### PR DESCRIPTION
**Pull request by fdintino**
I stumbled upon this while trying to write some unit tests that required StandaloneImage instances. I think `default_open` was intended to be `default_storage.open`.

### CODE REVIEW

File: cropduster/standalone/models.py [hunk 1]
```diff
@@ -54,17 +54,18 @@ class StandaloneImage(models.Model):
...
     def save(self, **kwargs):
-        if not self.md5:
+        if not self.md5 and self.image:
             md5_hash = hashlib.md5()
-            with default_open(self.image.path, mode='rb') as f:
+            with self.image.related_object.image as f:
+                f.open()
                 md5_hash.update(f.read())
-            self.md5 = md5_hash.digest()
+            self.md5 = md5_hash.hexdigest()
```
Comment (lorenries) [inline cropduster/standalone/models.py:70]: why the change from `digest` to `hexdigest`?

**Review by lorenries — APPROVED** [review]
lgtm 👍🏻, one (non-blocking) question

File: cropduster/standalone/models.py (hunk 1 above)
Comment (fdintino) [reply cropduster/standalone/models.py:70]: `.digest()` actually returns bytes, but what we want is the hexadecimal md5 so we can save it in a CharField. It's also what we use elsewhere in cropduster, e.g.:

https://github.com/theatlantic/django-cropduster/blob/c2168c36f34922b509a98e3dc667441799389916/cropduster/views/forms.py#L88
